devel/cvs-fast-export/Portfile

```diff
@@ -37,7 +39,7 @@ depends_build       port:flex \
 license_noconflict  bison openssl
 
 patchfiles          patch-Makefile.diff
-patchfiles-append   CLOCK_REALTIME.patch
+#patchfiles-append   CLOCK_REALTIME.patch
```

Comment:
Just delete the line, no need to uncomment :)

Reply:
Done.